api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -85,11 +85,10 @@ private ClusterRequest createFirecloudClusterRequest(
     // i.e. is NEW or MIGRATED
     if (!workspace.getBillingMigrationStatusEnum().equals(BillingMigrationStatus.OLD)) {
       customClusterEnvironmentVariables.put(
-          WorkbenchConstants.CDR_VERSION_CLOUD_PROJECT,
-          workspace.getCdrVersion().getBigqueryProject());
-      customClusterEnvironmentVariables.put(
-          WorkbenchConstants.CDR_VERSION_BIGQUERY_DATASET,
-          workspace.getCdrVersion().getBigqueryDataset());
+          WORKSPACE_CDR,
+          workspace.getCdrVersion().getBigqueryProject()
```

Comment:
nit: I feel like `String.format()`, or better yet a function to build the variable value, would be clearer.